src/pages/storage/StoragePoolOptionLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+import { LxdStoragePool } from "types/storage";
+import StoragePoolSize from "./StoragePoolSize";
+
+interface Props {
+  pool: LxdStoragePool;
+}
+
+const StoragePoolOptionLabel: FC<Props> = ({ pool }) => {
+  return (
+    <div className="label">
+      <span key="name" title="Name" className="resource u-truncate">
+        {pool.name || "-"}
+      </span>
+      <span key="driver" title="Driver" className="resource-driver u-truncate">
+        {pool.driver || "-"}
+      </span>
+      <span key="usage" title="Usage" className="resource-usage u-truncate">
+        <StoragePoolSize key={pool.name} pool={pool} shorthand_text={true} />
```

Comment:
There is a design problem with using the size graph here:

![image](https://github.com/user-attachments/assets/aa67e924-4b07-443b-adfc-3aee3f42a33f)

The highlight color and the usage color are matching and leaving no contrast to understand. Can you find a better solution?

Reply:
Solution discussed in call: Conditionally removing the meter bar as having the same bar represented with different colour across the UI would be inconsistent, and the colour cannot be changed from Blue and is it a Vanilla colour.